src/rules/flake8_pytest_style/rules/unittest_assert.rs

```diff
@@ -265,12 +265,15 @@ impl UnittestAssert {
                     .get("expr")
                     .ok_or_else(|| anyhow!("Missing argument `expr`"))?;
                 let msg = args.get("msg").copied();
-                let bool = create_expr(ExprKind::Constant {
-                    value: Constant::Bool(matches!(self, UnittestAssert::True)),
-                    kind: None,
-                });
-                let expr = compare(expr, Cmpop::Is, &bool);
-                Ok(assert(&expr, msg))
+                if matches!(self, UnittestAssert::True) {
+                    let unary_expr = create_expr(ExprKind::UnaryOp {
+                        op: Unaryop::Not,
+                        operand: Box::new(create_expr(expr.node.clone())),
+                    });
+                    Ok(assert(&unary_expr, msg))
+                } else {
+                    Ok(assert(expr, msg))
```

Comment:
I think you want to swap these

```suggestion
                    Ok(assert(expr, msg))
                } else {
                    let unary_expr = create_expr(ExprKind::UnaryOp {
                        op: Unaryop::Not,
                        operand: Box::new(create_expr(expr.node.clone())),
                    });
                    Ok(assert(&unary_expr, msg))
```

Reply:
correct, thanks! I adjusted this.